### ISSUE TITLE
Search input has no label

### ISSUE THREAD

**Issue by andest01** [troutspotr/troutspotr-frontend#73]
<img width="763" alt="screen shot 2017-08-29 at 4 27 45 pm" src="https://user-images.githubusercontent.com/1424223/29845056-0b05a638-8cd7-11e7-997b-9b6a32250c10.png">

**Comment by andest01** [comment]
https://css-tricks.com/float-labels-css/
